apps/dashboard/src/components/primitives/form/form.tsx

```diff
@@ -6,8 +6,9 @@ import { Controller, ControllerProps, FieldPath, FieldValues, FormProvider } fro
 import { cn } from '@/utils/ui';
 import { Label } from '@/components/primitives/label';
 import { cva } from 'class-variance-authority';
-import { RiInformationFill } from 'react-icons/ri';
 import { FormFieldContext, FormItemContext, useFormField } from './form-context';
+import { RiErrorWarningFill, RiInformationFill } from 'react-icons/ri';
+import { BsFillInfoCircleFill } from 'react-icons/bs';
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** What is our approach for icons for the dashboard?
I see we are also creating react svg files from figma.‏

Reply:
This was the closest I could find. I generally thought we were using remix icons, but it seems that it's not the case.

Reply:
we are also using icons like [this](https://github.com/novuhq/novu/blob/next/apps/dashboard/src/components/icons/notification-5-fill.tsx) by copying from the figma directlt

Reply:
Yeah but some match icons in `react-icons`, so there's no need to do that. Do you think I should copy the svg in this case?

Reply:
No i think it's fine, was just curious about how we are using the icons in the new dashboard

Reply:
only these are the custom icons: https://www.figma.com/design/iOS9MnuJjLR7avdSIHemFy/novu-v3.0-Design-System-%E2%9A%A1%EF%B8%8F?node-id=125-263&node-type=text&m=dev
the rest are remix icons